### PR TITLE
bug/19297 - Bug fix

### PR DESCRIPTION
**Pull request by pris54**
Trim spaces in group name field for better validation.

### CODE REVIEW

File: admin/assets/javascripts/global-scripts.js [hunk 1]
```diff
@@ -382,7 +382,7 @@ $(function () {
      * @returns {boolean}
      */
     isGroupNameComplete: function () {
-      var elName = $('input#name').val()
+      var elName = $('input#name').val().trim()
```
Comment (jon-shipley) [inline admin/assets/javascripts/global-scripts.js]: I suspect that this won't work in IE8.

Comment (pris54) [reply admin/assets/javascripts/global-scripts.js]: 🙄  I'll test it on IE8. However this is an edge case and it's not a critical fix because we still get a server side error on the screen.

Comment (pris54) [reply admin/assets/javascripts/global-scripts.js]: @jon-shipley - Tests passed and @sinhaamit1980 is happy with this PR. Can we please get this approved?

Comment (GuyHarwood) [reply admin/assets/javascripts/global-scripts.js]: Are we even supporting IE8. I can never remember 🤔

Comment (jon-shipley) [reply admin/assets/javascripts/global-scripts.js]: @GuyHarwood - functional support for IE8-10

Comment (jon-shipley) [reply admin/assets/javascripts/global-scripts.js]: @pris54 

Why is this a client-side fix and not server side?
We need to support IE functionally and calling methods that may not be implemented doesn't work for me.   You could use Jquery.trim() though?

Comment (GuyHarwood) [reply admin/assets/javascripts/global-scripts.js]: In that case safest, and likely easiest would be as Jon says, server side

**Comment by pris54** [comment]
@GuyHarwood @jon-shipley I'm not sure you understand what this bug is about... I can demo it to you if it helps.

File: admin/assets/javascripts/global-scripts.js (hunk 1 above)
Comment (jon-shipley) [reply admin/assets/javascripts/global-scripts.js]: @pris54 

Fix it?

screen shot 2018-02-14 at 14 00 50

**Comment by pris54** [comment]
@GuyHarwood can I please get this PR approved? I've done the changes requested.
